### ISSUE TITLE
Add NextCloud Notes app

### ISSUE THREAD

**Issue by duxovni** [mail-in-a-box/mailinabox#1525]
Back when MIAB was still using ownCloud, there was a pull request to add the ownCloud Notes app, but it was rejected because we were about to migrate away from ownCloud anyway. Now that we're using NextCloud, could we add the NextCloud Notes app?

**Comment by pandruszkow** [comment]
Mail in a Box's core feature is providing a reliable, secure one-click email server, which incidentally runs NextCloud for some back-ends. If adding the Notes app exposes more of the underlying NextCloud, then Mail in a Box turns into a mail+Nextcloud offering, and now both would have to be maintained and there would be an additional dependency on NextCloud in case we want to migrate in the future after all.

I would personally run NextCloud on a different server and leave my Mail in a Box instance entirely separate from non-email uses as much as I can. You could always federate your MiaB NextCloud instance and the other NextCloud instance together, or use one as a WebDAV target for syncing.

**Comment by JoshData** [comment]
Exactly. We really only have Nextcloud so that the box provides a way to store contacts, which is an important part of email functionality and even email security. I added the calendar because, well, it was the one other service I would need personally in order to leave the Google ecosystem.

**Comment by insideTheFlux** [comment]
A bit off-topic.

How have you found the process of leaving Google @JoshData ? Was it easy? 

I'm still struggling with it a bit even though I've been running Miab for a year.

**Comment by JoshData** [comment]
Well, I use Google all the time... Maybe leaving the ecosystem was an exaggeration. I love Google Docs and use it all the time (especially for work). My phone is always logged into Google (at least for the benefit of knowing where my phone is). But I'm not dependent on it (at least I don't think I am). If Google disappeared, I'd like to think it might take me a few days to notice.

**Comment by pandruszkow** [comment]
I actually second the bit about leaving Google's ecosystem too. It's hard to imagine how cool having your own synced calendar is, until you actually have it.
